exporter/kafkaexporter/exporter.go

```diff
@@ -0,0 +1,102 @@
+package kafkaexporter
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"sync"
+	"text/template"
+
+	"github.com/IBM/sarama"
+	"github.com/thomaspoignant/go-feature-flag/exporter"
+	"github.com/thomaspoignant/go-feature-flag/utils/fflog"
+)
+
+type Exporter struct {
+	// Format is the output format you want in your exported file.
+	// Available format are JSON, CSV and Parquet.
+	// Default: JSON
+	Format string
+
+	// CsvTemplate is used if your output format is CSV.
+	// This field will be ignored if you are using another format than CSV.
+	// You can decide which fields you want in your CSV line with a go-template syntax,
+	// please check internal/exporter/feature_event.go to see what are the fields available.
+	// Default:
+	// {{ .Kind}};{{ .ContextKind}};{{ .UserKey}};{{ .CreationDate}};{{ .Key}};{{ .Variation}};{{ .Value}};{{ .Default}}\n
+	CSVTemplateString string
```

Comment:
I am not sure if I would accept CSV to push the message in Kafka.
I would start by pushing in JSON only, and if we want to have different formats I would for `avro` which is probably more common for the kafka users.